### DISCUSSION TITLE
Query latlng within bounds

### DISCUSSION THREAD

**Discussion by iChintanSoni** [sequelize/sequelize#14612] [Help & Questions]
### Describe the feature you'd like to see implemented

Suppose, I have a table that stores information about User as id, name, email, latitude & longitude. I have bounds information from Google maps using [getBounds()](https://developers.google.com/maps/documentation/javascript/reference/map#Map.getBounds). Now I wish to query users whose latitude and longitude lies within bounds. Not sure if Sequelize already supports this or any pointers in this direction.

### Describe why you would like this feature to be added to Sequelize
It will help make geolocation queries as explained in the above description.

### Is this feature dialect-specific?

- [ ] No. This feature is relevant to Sequelize as a whole.

### Would you be willing to resolve this issue by submitting a Pull Request?

- [ ] No, I don't have the time, and I understand that I will need to wait until someone from the community or maintainers is interested in implementing my feature.

---

*Indicate your interest in the addition of this feature by adding the 👍 reaction. Comments such as "+1" will be removed.*

**Comment by ephys** [comment]
I'm moving this to a discussion because this is already possible:

```js
import { Op } from 'sequelize';

const ne = bounds.getNorthEast();
const sw = bounds.getSouthWest();

await User.findAll({
  where: {
    latitude: { [Op.between]: [sw.lat(), ne.lat()] },
    longitude: { [Op.between]: [sw.lng(), ne.lng()] },
  },
});
```

You'll need to double check, I may have mixed the coordinates up, and it depends on how you store the lat/long, but this is basically how you'd do this

**Reply by iChintanSoni** [reply]
Thank you @ephys I will try this and let you know!
